src/index.js

```diff
@@ -1,46 +1,87 @@
-// import _ from 'lodash';
+// import _, { isLength } from 'lodash';
 import './style.css';
 
-const toDoTasks = [
-  {
-    description: 'index10',
-    completed: true,
-    index: 10,
-  },
-  {
-    description: 'index3',
-    completed: false,
-    index: 3,
-  },
-  {
-    description: 'index2',
-    completed: true,
-    index: 2,
-  },
-  {
-    description: 'index1',
-    completed: true,
-    index: 1,
-  },
-  {
-    description: 'index15',
-    completed: false,
-    index: 15,
-  },
-  {
-    description: 'index25',
-    completed: true,
-    index: 25,
-  },
-];
-
-const orderToDoTasks = toDoTasks.sort((a, b) => (a.index > b.index ? 1 : -1));
-
+let description;
+let completed;
+let index;
+let bookList = JSON.parse(localStorage.getItem('bookList')) || [];
 let bookgenerator = '';
-const listaUl = document.getElementById('list-ul');
+const taskList = document.getElementById('taskList');
+let bookListDeleted = [];
+const input = document.getElementById('myInput');
+
+class Book {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+function getTask() {
+  description = input.value;
+  completed = false;
+  index = bookList.length + 1;
+}
+
+function addBook(description, completed, index) {
+  const newBook = new Book(description, completed, index);
+  bookList.push(newBook);
+  localStorage.setItem('bookList', JSON.stringify(bookList));
+}
 
-for (let i = 0; i < orderToDoTasks.length; i += 1) {
-  bookgenerator += `<li >"${orderToDoTasks[i].description}"</li>`;
+function display(bookList) {
+  bookgenerator = '';
+  for (let i = 0; i < bookList.length; i += 1) {
+    bookgenerator += `<div>
+    <input type="text" id="${bookList[i].index * 10}" placeholder="${bookList[i].description}">
+    <button type="button" id="${bookList[i].index}">Remove</button></div>`;
+  }
+  taskList.innerHTML = bookgenerator;
 }
 
-listaUl.innerHTML = bookgenerator;
+function remove(event) {
+  const remove = event.target.closest('BUTTON');
```

Comment:
- [ ]  Good job to implement the new functionalities but as suggested, you should create file for new functionalities it means that the methods that you've implemented for this milestone should be in their own file. this will make your project more organized and clean.